docs/UserGuide.md

```diff
@@ -161,21 +161,23 @@ Example:
 
 ###  Search for a Contact: `search`
 
-Search for contacts using tags or other information.
+Search for contacts using partial information.
 
-Format: `search [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…`
+Format: `search [CONDITION] [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…`
```

Comment:
May I know what are the `CONDITION` there are?

Reply:
the `and` and `or`